### PR TITLE
✨(edxapp) add Fonzie to OEE dependencies

### PR DESCRIPTION
**Pull request by rmoch**
## Purpose

Add Fonzie to add access control to instructor dashboard CSV export files, and to future edx-platform evolutions

### CODE REVIEW

File: config/lms/root_urls.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+from __future__ import absolute_import, unicode_literals
+
+from django.conf.urls import include, url
+
+from lms.urls import urlpatterns  # pylint: disable=import-error
+
+
+urlpatterns += [
+    # Fonzie API urls
+    url(r"^api/", include("fonzie.urls", namespace="fonzie")),
```
Comment (jmaupetit) [inline config/lms/root_urls.py]: There is no edx urls prefixed with `api`?

Comment (rmoch) [reply config/lms/root_urls.py]: Indeed edX has many `^api/whatever` routes, however our `^api/v1.0/{status,acl}` will not interfer.
We may rename this `^fonzie/api/*` 
WDYT ?

Comment (jmaupetit) [reply config/lms/root_urls.py]: If our URK schema does not interfere, you can let it as is.

File: config/lms/docker_run_production.py [hunk 2]
```diff
@@ -829,8 +832,20 @@
     "GRADES_DOWNLOAD_ROUTING_KEY", default=HIGH_MEM_QUEUE
 )
 
+# Fonzie API endpoint to add access control on instructor dashboard
+# CSV export files (activated by features ENABLE_GRADE_DOWNLOADS and
+# ALLOW_COURSE_STAFF_GRADE_DOWNLOADS)
+
```
Comment (jmaupetit) [inline config/lms/docker_run_production.py]: Empty line to remove

**Pull request merged**